src/__tests__/requirement13.test.js

```diff
@@ -1,34 +1,32 @@
 import React from 'react';
 import { render, screen, waitFor, fireEvent } from '@testing-library/react';
 import App from '../App';
-import * as api from '../services/api';
-import mockedCategoriesResult from '../__mocks__/categories';
-import mockedQueryResult from '../__mocks__/query';
-
-jest.mock('../services/api');
-api.getCategories.mockImplementation(
-  () => Promise.resolve(mockedCategoriesResult),
-);
-api.getProductsFromCategoryAndQuery.mockImplementation(
-  () => Promise.resolve(mockedQueryResult),
-);
+import mockFetch from '../__mocks__/mockFetch';
 
 describe(`13 - Mostre junto ao ícone do carrinho a quantidade de produtos dentro dele, em todas as telas em que ele aparece`, () => {
   it('Vê a quantidade de produtos no carrinho da tela de listagem', async () => {
+
+    jest.spyOn(global, 'fetch').mockImplementation((url) => {
+      return mockFetch(url);
+    });
     render(<App />);
-    await waitFor(() => expect(api.getCategories).toHaveBeenCalled());
+    await waitFor(() => expect(global.fetch).toHaveBeenCalled());
     fireEvent.click(screen.getAllByTestId('category')[0]);
-    await waitFor(() => expect(api.getProductsFromCategoryAndQuery).toHaveBeenCalled());
+    await waitFor(() => expect(global.fetch).toHaveBeenCalled());
```

Comment:
Aqui também :grimacing: 

```suggestion
    await waitFor(() => expect(global.fetch).toHaveBeenCalledTimes(2));
```